cmd/gardenlet/app/migration.go

```diff
@@ -165,3 +175,38 @@ func syncBackupSecretRefAndCredentialsRef(backup *gardencorev1beta1.Backup) {
 	// - both fields are set -> let the validation check if they are correct
 	// - credentialsRef refer to WorkloadIdentity -> secretRef should stay unset
 }
+
+// TODO(@aaronfern): Remove this after v1.122 is released
```

Comment:
Sorry, we won't be able to include the PR in the 1.120 release. Could you adapt the TODOs (in other places as well):
```suggestion
// TODO(@aaronfern): Remove this after v1.123 is released
```